DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
 Any help to fix this is very welcome!
 
+#### Integration with VSCode and Delve
+
+If you would like to use the delve debugger for Go with VSCode to debug `gapis`, You can follow these steps
+
+1. Complete the fusing steps.
+
+2. Create a launch.json under the project directory `Ctrl + Shift + P` and `Debug: Open launch.json`
+
+3. Paste this as one of the configuration. This will ensure that you have a setup for attaching to Delve when you launch the project.
+```
+{
+      "name": "Attach to Delve",
+      "type": "go",
+      "request": "attach",
+      "mode": "remote",
+      "apiVersion": 2,
+      "remotePath": "gapis/",
+      "cwd": "${workspaceFolder}/src/github.com/google/gapid/gapis",
+      "dlvLoadConfig": {
+          "followPointers": true,
+          "maxVariableRecurse": 1,
+          "maxStringLen": 120,
+          "maxArrayValues": 120,
+          "maxStructFields": -1
+      },
+      "host": "127.0.0.1",
+      "port": 1234,
+    },
+```
+
+4. Start delve in headless mode at your gapid folder. `dlv exec --headless --listen=127.0.0.1:1234 --api-version 2 ./bazel-bin/pkg/gapis` This will allow you to use port 1234 to connect to delve from VSCode.
+
+5. Now launch your project(on Linux with `F5`) and make sure `Attach to Delve` is selected as the launch configuration.
+
+6. Now you can put breakpoints in VScode and debug `gapis` on UI instead of command line. Enjoy your debugging :)
```

Comment:
6. You should now be able to use the VSCode debugging UI to interact with delve. This let you add breakpoints, step through the code, etc. More information about VSCode debugging UI is available [here](https://code.visualstudio.com/docs/editor/debugging).

Reply:
I am not favor of giving external links because they may be outdated, changed and we need to keep track of them. I guess it is fairly easy to find it and if someone is following these steps they are probably already aware of it.